internal/platform/ipu.go

```diff
@@ -58,8 +58,10 @@ func (pi *IntelDetector) IsDpuPlatform() (bool, error) {
 }
 
 func (pi *IntelDetector) VspPlugin(dpuMode bool, vspImages map[string]string, client client.Client) *plugin.GrpcPlugin {
-	intelVspImage := vspImages["IntelVspImage"]
-	return plugin.NewGrpcPlugin(dpuMode, client, plugin.WithVspImage(intelVspImage))
+	template_vars := plugin.CreateVspImageVars(vspImages["IntelDetector"])
```

Comment:
Shouldn't this be "IntelVspImage"?

Reply:
of course... thanks.

(reminds me to when I suggested not to use string literals, because they are error prone).